From f2f94d5b656586fef01cad0af9cc8213b7348047 Mon Sep 17 00:00:00 2001
From: David Spickett <david.spickett@linaro.org>
Date: Mon, 14 Feb 2022 13:36:51 +0000
Subject: [PATCH] [compiler-rt][xray] Disable fdr-reinit test on Arm

This test is still seemingly randomly segfaulting on Arm:
https://lab.llvm.org/buildbot/#/builders/178/builds/1547

Though it seems to fail earlier in the test than on AArch64.
Investigation continues.
---
 compiler-rt/test/xray/TestCases/Posix/fdr-reinit.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/compiler-rt/test/xray/TestCases/Posix/fdr-reinit.cpp b/compiler-rt/test/xray/TestCases/Posix/fdr-reinit.cpp
index 42e7fa24d54e..faf1372d61bd 100644
--- a/compiler-rt/test/xray/TestCases/Posix/fdr-reinit.cpp
+++ b/compiler-rt/test/xray/TestCases/Posix/fdr-reinit.cpp
@@ -1,5 +1,5 @@
-// This will sometimes segfault on the AArch64 bots
-// UNSUPPORTED: aarch64
+// This will sometimes segfault on the AArch64 and Arm bots
+// UNSUPPORTED: aarch64, arm
 // RUN: %clangxx_xray -g -std=c++11 %s -o %t
 // RUN: rm xray-log.fdr-reinit* || true
 // RUN: XRAY_OPTIONS="verbosity=1" %run %t
-- 
2.39.3

